Application/EdFi.Common/Extensions/StringExtensions.cs

```diff
@@ -86,6 +86,18 @@ public static string[] SplitCamelCase(this string text)
         /// <param name="delimiters">Additional delimiters to use as word breaks.</param>
         /// <returns>The text processed for display.</returns>
         public static string NormalizeCompositeTermForDisplay(this string compositeTerm, params char[] delimiters)
+        {
+            return NormalizeCompositeTermForDisplay(compositeTerm, ToMixedCase, delimiters);
+        }
+
+        /// <summary>
+        /// Breaks the supplied text into individual words using mixed-casing conventions augmented by delimiters.
+        /// </summary>
+        /// <param name="compositeTerm">The text to process for display.</param>
+        /// <param name="applyCasing">A function that can be used to process each term before inclusion in the final text.</param>
+        /// <param name="delimiters">Additional delimiters to use as word breaks.</param>
+        /// <returns>The text processed for display.</returns>
+        public static string NormalizeCompositeTermForDisplay(this string compositeTerm, Func<string, string> applyCasing, params char[] delimiters)
```

Comment:
This enhancement was to allow this function to be used to break apart the custom view-based authorization strategy name into something that can be displayed as a "hint" when an authorization fails. Rather than using title-casing across every word, the custom view-based authorization uses a word list based on a set of common prepositions to use casing that is more consistent with a typical sentence.